examples/tutorials/asr_inference_with_ctc_decoder_tutorial.py

```diff
@@ -159,6 +158,23 @@
 #
 
 
+######################################################################
+# Language Model
+# ~~~~~~~~~~~~~~
+#
+# A language model can be used in decoding to improve the results, by
+# factoring in a language model score that represents the likelihood of
+# the sequence into the beam search computation. Below, we outline the
+# different forms of language models that are supported for decoding.
+# 
+
+######################################################################
+# No Language Model
+# ^^^^^^^^^^^^^^^^^
+#
+# To train without a language model, set `lm=None` when initializing
```

Comment:
Without a language model, is decoder object still something trainable?
“To create a decoder instance without a language model” sounds more intuitive. (I’m reviewing from my phone and didn’t check the scrounging context that is not changed so I might be missing something’s here